iglo/league/models.py

```diff
@@ -859,7 +862,7 @@ def create_bye_game(self, group: Group, round: Round, member: Member):
     def get_immediate_games(self):
         now = datetime.datetime.now()
         start = now + datetime.timedelta(hours=2)
-        end = now + datetime.timedelta(days=8)
+        end = now + datetime.timedelta(days=6, hours=23)
```

Comment:
why is this changed in that pr? It seems there's some divergence between branches fixed in this pr

Reply:
It was devel->egd merge from old devel I think. 
Is it fixed?